test/rulesets/LinearAlgebra/factorization.jl

```diff
@@ -12,6 +12,133 @@ function FiniteDifferences.to_vec(x::Val)
 end
 
 @testset "Factorizations" begin
+    @testset "lu decomposition" begin
+        # avoid implementing to_vec(::LU)
```

Comment:
why are we avoiding this?

Reply:
I had thought we were avoiding type-piracy, but I see now we define `to_vec` for `Cholesky` in this file. I have added `to_vec` for `LU` now.